# 
# Backport r1031526 from trunk to 4.3:
# Add the KMAIL_SUPPORTED definition only if KMail is compiled.
#
 
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(révision 1031531)
+++ CMakeLists.txt	(révision 1031532)
@@ -159,8 +159,11 @@
     # The following components depends on QGpgME.
     macro_optional_add_subdirectory(kmail)
     # If kmail is compiled, KMAIL_SUPPORTED is true (used in several places)
-    macro_bool_to_01(BUILD_kmail KMAIL_SUPPORTED)
-    add_definitions(-DKMAIL_SUPPORTED)
+    if(BUILD_kmail)
+      set(KMAIL_SUPPORTED TRUE)
+      add_definitions(-DKMAIL_SUPPORTED)
+    endif(BUILD_kmail)
+
     macro_optional_add_subdirectory(kmailcvt)
     macro_optional_add_subdirectory(ksendemail)
     macro_optional_add_subdirectory(korganizer)
